File: Class 2/Home Work/Leet Code 27 Remove Element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        i = 0
#Check if value exists
        while(val in nums):
#Remove value at position
            if(nums[i] == val):
                nums.pop(i)
                i-=1 #Step back once since the size has reduced.
            i+=1

File: Class 2/Home Work/LeetCode 27 Remove Element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int i;
        int s = 0;
        int n = nums.size();

        if(n==0){
            return 0;
        }
        for(i=0;i<n;i++){
            if(nums[i]!=val){
                nums[s++]=nums[i];
            }
        }
        return s;
    }
};
